.github/workflows/trivy.yml

```diff
@@ -0,0 +1,107 @@
+name: Scheduled Trivy Scan
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 10 * * 1'
+
+jobs:
+  trivy-sbom:
+    name: Generate SBOM via Trivy 
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Run Trivy in GitHub SBOM mode and submit results to Dependency Graph
+        uses: aquasecurity/trivy-action@0.11.2
+        with:
+          scan-type: 'fs'
```

Comment:
does this find any dependencies, as they wont have be installed yet?
I expect this will produce a different result to the docker image checks, as it definitely wont be finding some of the dependencies tucked away deep inside of meteor

Reply:
You're right, I was fooled by testing it locally where I have all the node_modules. I'll just add SBOM generation from a scan of whatever image is latest.